src/pages/SingleArticle.jsx

```diff
@@ -0,0 +1,4 @@
+function SingleArticle() {
+
+}
+export default SingleArticle;
```

Comment:
Don't really need this yet
Try stick to the individual ticket and what its asked for.